packages/recommend-react/src/__tests__/useFrequentlyBoughtTogether.test.ts

```diff
@@ -37,9 +37,99 @@ describe('useFrequentlyBoughtTogether', () => {
           facetFilters: ['test'],
         },
         transformItems: (items) => items,
-      });
+      })
+    );
 
-      expect(recommendations).toEqual([hit]);
+    await waitFor(() => {
+      expect(result.current.recommendations).toEqual([hit]);
     });
   });
+
+  test('returns FBT recommendations from initialState', async () => {
+    const { recommendClient } = createMockedRecommendClient();
+
+    const { result } = renderHook(
+      () =>
+        useFrequentlyBoughtTogether({
+          indexName: 'test',
+          recommendClient,
+          threshold: 0,
+          objectIDs: ['testing'],
+          queryParameters: {
+            facetFilters: ['test'],
+          },
+          transformItems: (items) => items,
+          initialState,
+        }),
+      {
+        wrapper: React.StrictMode,
+      }
+    );
+
+    await waitFor(() => {
+      expect(result.current.recommendations).toEqual(
+        initialState.recommendations
+      );
```

Comment:
I don't think is necessary as this feature will be used for React SSR 🤔

Reply:
It is because it actually reveals a bug right now: https://github.com/algolia/recommend/blob/04f0380be41374298eb2e186c4e82dc024cb0f99/packages/recommend-react/src/useFrequentlyBoughtTogether.ts#L27

The status is `loading` although it should be `idle` when `initialState` is provided.

Reply:
ah right, will update the status accordingly

Reply:
there's a de-sync in the comments: 
"I don't think is necessary as this feature will be used for React SSR 🤔" -> in response to "Do you plan to also add this feature to @algolia/recommend-js?"

Reply:
update: https://github.com/algolia/recommend/pull/109/commits/64f060fd04ce1c559d047d98f4d5fa10e827f1c1